src/main/java/org/apache/sysml/runtime/instructions/cp/CtableCPInstruction.java

```diff
@@ -40,7 +40,7 @@
 	private final boolean _isExpand;
 	private final boolean _ignoreZeros;
 
-	private CtableCPInstruction(CPOperand in1, CPOperand in2, CPOperand in3, CPOperand out,
```

Comment:
Revert these changes to the visibility if you call the block operations directly.